### ISSUE TITLE
Statistics.Histogram.Histogram Needs Testing

### ISSUE THREAD

**Issue by HenryGeorgist** [HydrologicEngineeringCenter/HEC-FDA#114]
Create a unit test for a Statistics.Histogram.Histogram, follow the testing strategy we generally followed here:
https://github.com/HydrologicEngineeringCenter/go-statistics/blob/master/data/inline-histogram_test.go

I liked many of the tests in the go library that @rnugent3 created, test adding values out of range above and below. Test pulling data out and verify it meets expectations. 

Specifically, I would like to see a Standard normal distribution created, and feed a Histogram with it. 
Then add a Log Pearson III into it in another test.

**Comment by rnugent3** [comment]
When you say you would like to see a standard Normal distribution created, you're saying that you'd like to see data generated using the standard Normal distribution, and have that data fed into a histogram?

**Comment by HenryGeorgist** [comment]
Yes, that is exactly what I mean.

**Comment by rnugent3** [comment]
sounds good, thank you.

**Comment by rnugent3** [comment]
@HenryGeorgist please take a look at the work I started and critique at your leisure.  

Test class is here: https://github.com/HydrologicEngineeringCenter/HEC-FDA/blob/07afcf4a0f43fefc9468df87710bb51b6510e73b/StatisticsTests/Histograms/HistogramTests.cs#L14

Some notes: 
1. Something funny is going on with the standard deviation and variance, I have to look into that. After I do so, I'll get the standard deviation and variance tests in.
2. I plan to build out the test data for all tests
3. Once I fix the problems with the normally distributed histogram tests, I will move onto the Log Pearson Type III. 
4. I am concerned about how sensitive these tests are to sample size. Not sure if there is a better way to approach tests. 

Thanks.

**Comment by rnugent3** [comment]
On Point 4 - it just occurred to me that I can determine n based on convergence, which should solve that problem.

**Comment by HenryGeorgist** [comment]
the code looks good - next time it would be great to review that through a pull request instead of commenting on commits directly i think.

**Comment by HenryGeorgist** [comment]
yeah i think so, Pull requests in github have two nice features, 

1. marking a pr as "draft" to indicate it is not yet ready for merge/review
2. re-requesting a review

So if you want a reviewer to do a quick check on the work, request a review and specify it is work in progress WIP and that you want a general input on a design question. That way the PR will have all of the back and forth and we wont have to search for that info in the commit messages (at least i think that makes it easier).
